app/models/vote.rb

```diff
@@ -0,0 +1,7 @@
+class Vote < ActiveRecord::Base
+  belongs_to :votable, polymorphic: true
+  validates :votable_id, :votable_type, :user_id, :value, presence: true
```

Comment:
you can validate just the presence of `votable` instead of both votable attributes here (as long as you've defined `votable` as an association)